core/README.md

```diff
@@ -3,58 +3,51 @@
 <img src="https://user-images.githubusercontent.com/12449023/166088435-a9fcc4c7-f51d-443a-b1fd-9fe96a204f77.png" >
 </p>
 
-# The Industrial Ontologies Foundry Core Ontology
+# The IOF Core Ontology
 
-Version 1 Beta, 6-May-2022
+Version  1, Release 2022-02
 
 *The IOF is an international standards group that operates under the auspices of the Open Applications Group.*
 
 # Introduction
 
-The IOF’s mission is to create a suite of ontologies intended to support digital manufacturing by facilitating cross-system integration both within the factory and across an enterprise, in commerce between suppliers, manufacturers, customers and other trading partners, and throughout the various stages of the product life cycle. The IOF Core Ontology resides at the top of this suite from an architectural perspective and contains terms found in a number of operational areas of manufacturing. These common terms appear, or are anticipated to appear, in two or more of the ontologies of the suite. Additionally, as the architectural approach chosen by the IOF is to base all of its ontologies on a single foundational or top-level ontology – for which the IOF chose the Basic Formal Ontology or BFO – the Core Ontology contains a number of intermediate-level terms that derive from BFO and from which common industry terms are in turn derived. Such intermediate-level terms are most often domain independent – meaning they are found in other industries and domains, such as in the banking, insurance, and healthcare industries, or in the sciences, as in the physics, chemistry and biology domains. 
+The IOF Core Ontology (Core) is a formal ontology that provides a common vocabulary for the industrial domains. Core is a "mid-level" ontology in that it resides at the top of the suite of ontologies the IOF seeks to introduce to fulfill its mission of supporting digital manufacturing by standardizing industrial terminology and improving consistency and interoperability across many operational areas of manufacturing and the product life cycle. As a mid-level ontology, Core contains terms used by, or anticipated to be used by, a plurality of ontologies in the suite. Additionally, as the IOF bases all of its ontologies on a single foundational or top-level ontology – for which the IOF has chosen the Basic Formal Ontology or BFO – the Core ontology contains generic terms found in domains beyond industry that build upon BFO and provide a consistent foundation for introducing industry-specific terminology. 
```

Comment:
I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

Reply:
The beginning looks good. I would still move the Getting Involved section to the top level README. It does not seem to be relevant to core.

Reply:
Deleted the Getting Involved Section from here since it is now covered in root. Have another look at latest commit and please approve so we can publish.